runtime/battery-station/src/parameters.rs

```diff
@@ -288,6 +288,35 @@ parameter_types! {
     /// Pallet identifier, mainly used for named balance reserves.
     pub const TreasuryPalletId: PalletId = PalletId(*b"zge/tsry");
 
+    // Bounties
+    /// The amount held on deposit for placing a bounty proposal.
+    pub const BountyDepositBase : Balance = 10 * BASE;
+    /// The delay period for which a bounty beneficiary need to wait before claim the payout.
+    pub const BountyDepositPayoutDelay : BlockNumber = 3 * BLOCKS_PER_DAY;
+
+    /// Bounty duration in blocks.
+    pub const BountyUpdatePeriod : BlockNumber = 14 * BLOCKS_PER_DAY;
+
+    /// The curator deposit is calculated as a percentage of the curator fee.
+    ///
+    /// This deposit has optional upper and lower bounds with `CuratorDepositMax` and
+    /// `CuratorDepositMin`.
+    pub const CuratorDepositMultiplier : Permill = Permill::from_percent(50);
+
+    /// Maximum amount of funds that should be placed in a deposit for making a proposal.
+    pub const CuratorDepositMax : Balance = 200 * BASE;
+    /// Minimum amount of funds that should be placed in a deposit for making a proposal.
+    pub const CuratorDepositMin : Balance = 10 * BASE;
+    /// Minimum value for a bounty.
+    pub const BountyValueMinimum: Balance = BASE;
+
+    /// The amount held on deposit per byte within the tip report reason or bounty description.
+    pub DataDepositPerByte : Balance = BASE;
+    /// Maximum acceptable reason length.
+    ///
+    /// Benchmarks depend on this value, be sure to update weights file when changing this value
+    pub MaximumReasonLength : u32 = 256;
```

Comment:
I think this could also be increased, because it's the maximum bytes for a description of a bounty. 

https://github.com/paritytech/substrate/blob/58d3bc67b9595d0a26f5abba88fe4941a6e75b4a/frame/bounties/src/lib.rs#L820-L821

[Kusama uses `16384` bytes at the moment](https://github.com/paritytech/polkadot/blob/d4d648e5781b6fefd83b3a22bf5a3914e460fc3f/runtime/kusama/src/lib.rs#L680).